src/components/brands/EditorToolbar.jsx

```
@@ -0,0 +1,195 @@
+        newCursorPos = selectedText ? end + 4 : start + 2;
+        break;
+        
+      case 'italic':
+        newText = text.substring(0, start) + 
+                  `*${selectedText || 'italic text'}*` + 
+                  text.substring(end);
+        newCursorPos = selectedText ? end + 2 : start + 1;
+        break;
+        
+      case 'link':
+        const linkMarkdown = `[${linkText || selectedText || 'link text'}](${linkUrl})`;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)